frontend/src/components/MapView/LeftPanel/AnticipatoryActionPanel/index.tsx

```diff
@@ -0,0 +1,270 @@
+} from 'context/anticipatoryActionStateSlice';
+import { dateRangeSelector } from 'context/mapStateSlice/selectors';
+import { getFormattedDate } from 'utils/date-utils';
+import { DateFormat } from 'utils/name-utils';
+import HomeTable, { RowProps } from './HomeTable';
+import { AAIcons, StyledRadioLabel } from './utils';
+
+const buttons = [
+  { icon: GetApp, text: 'Assets' },
+  { icon: EditOutlined, text: 'Report' },
+  { icon: BarChartOutlined, text: 'Forecast' },
+];
+
+const links = [
+  { text: 'Group assumptions', href: 'google.com' },
+  { text: 'How to read this screen', href: 'google.com' },
+];
+
+function AnticipatoryActionPanel() {
+  const classes = useStyles();
+  const { t } = useSafeTranslation();
+  const RawAAData = useSelector(AnticipatoryActionDataSelector);
+  const MonitoredDistricts = [...new Set(RawAAData.map(x => x.district))];
+  // eslint-disable-next-line no-console
+  console.log(MonitoredDistricts);
+
+  const AAData = RawAAData.filter(
+    x => x.probability !== 'NA' && Number(x.probability) > Number(x.trigger),
+  );
+  // TODO: move this to redux state
+  const [selectedWindow, setSelectedWindow] = React.useState<string>('all');
+  const { startDate: selectedDate } = useSelector(dateRangeSelector);
+
+  const date = getFormattedDate(selectedDate, DateFormat.Default);
+
+  const AADateSameDate = React.useMemo(
+    () => AAData.filter(x => x.date === date),
+    [AAData, date],
+  );
+
+  const windows = React.useMemo(
+    () => [...new Map(AAData.map(x => [x.windows, x.windows])).keys()],
+    [AAData],
+  );
+
+  const setSev = React.useMemo(
+    () =>
+      AADateSameDate.filter(x => x.category === 'Severo' && x.phase === 'Set'),
+    [AADateSameDate],
+  );
+  const readySev = React.useMemo(
+    () =>
+      AADateSameDate.filter(
+        x => x.category === 'Severo' && x.phase === 'Ready',
+      ),
+    [AADateSameDate],
+  );
+  const setMod = React.useMemo(
+    () =>
+      AADateSameDate.filter(
+        x => x.category === 'Moderado' && x.phase === 'Set',
+      ),
+    [AADateSameDate],
+  );
+  const readyMod = React.useMemo(
+    () =>
+      AADateSameDate.filter(
+        x => x.category === 'Moderado' && x.phase === 'Ready',
+      ),
+    [AADateSameDate],
+  );
+
+  // TODO - LEVE is "MILD" and should be added as a new category, see Figma.
+  const setMild = React.useMemo(
+    () =>
+      AADateSameDate.filter(x => x.category === 'Leve' && x.phase === 'Set'),
+    [AADateSameDate],
+  );
+  const readyMild = React.useMemo(
+    () =>
+      AADateSameDate.filter(x => x.category === 'Leve' && x.phase === 'Ready'),
+    [AADateSameDate],
+  );
+
+  // eslint-disable-next-line no-console
+  console.log(setMild, readyMild);
+
+  // TODO: implement NA and NY
+  // NA means that no proba is above the trigger for that district
+  // NY means that the district is not monitored yet (no rows for the district)
+  const na = React.useMemo(() => [], []);
+  const ny = React.useMemo(() => [], []);
```

Comment:
@echaidemenos I think we need to revise this logic a bit and maybe do some pre-processing.
I added some explanations around what NA - No Action and NY - Not Yet Monitored mean.

I also added some filtering to only keep rows were proba > trigger

We also need to include Leve = "MILD" now, so just adding two rows.

And here is some additional info about the logic: to pass to “set”, the trigger must have been “ready” the preceding month. Once Severe Set is attained, the status stays the same for the rest of the window.

Can you propose a better way to do the data preparation? Also in general I think that logic such as language changes should happen as soon as we load the CSV so that the rest of the code does not have to worry about it. Wdyt?